plugin/pkg/scheduler/scheduler.go

```diff
@@ -188,7 +188,9 @@ func (s *Scheduler) scheduleOne() {
 	}
 
 	go func() {
-		defer metrics.E2eSchedulingLatency.Observe(metrics.SinceInMicroseconds(start))
+		defer func() {
```

Comment:
I updated my example code. Just like my example code says, without fix, the calculated time is wrong. And with fix, the time is right. And `metrics.E2eSchedulingLatency.Observe` wants the total duration of `scheduleOne` if my understanding is right.